app/Main.hs

```diff
@@ -42,21 +41,14 @@ import Project (
  )
 import Server (buildServer)
 
-import Metrics.Metrics qualified as Metrics
-import Metrics.Server (
-  MetricsServerConfig (
-    MetricsServerConfig,
-    metricsConfigHost,
-    metricsConfigPort
-  ),
-  runMetricsServer,
- )
+import Metrics qualified
 
 import Paths_hoff qualified (version)
 
 import Configuration qualified as Config
+import Control.Concurrent.STM (atomically)
+import Control.Concurrent.STM.TBQueue (isFullTBQueue)
```

Comment:
I think this block of imports is meant to have only qualified imports, and these should instead go below the `Control.Applicative` imports above.